pandas/tests/io/json/test_normalize.py

```diff
@@ -238,15 +238,16 @@ def test_non_ascii_key(self):
         tm.assert_frame_equal(result, expected)
 
     def test_missing_field(self, author_missing_data):
-        # GH20030: Checks for robustness of json_normalize - should
-        # unnest records where only the first record has a None value
+        # GH20030: Checks for robustness of json_normalize
```

Comment:
@WillAyd Can you verify this comment once again.

Reply:
Just the GH issue number is sufficient (like you have below)